frontend/src/pages/destructionlist/DestructionListCreate.tsx

```diff
@@ -12,19 +12,28 @@ import {
 } from "react-router-dom";
 
 import { loginRequired } from "../../lib/api/loginRequired";
+import { ZaaktypeChoice, listZaaktypeChoices } from "../../lib/api/private";
 import { PaginatedZaken, listZaken } from "../../lib/api/zaken";
 import { Zaak } from "../../types";
 import "./DestructionListCreate.css";
 
+export type DestructionListCreateContext = {
+  zaken: PaginatedZaken;
+  zaaktypeChoices: ZaaktypeChoice[];
+};
+
 /**
  * React Router loader.
  * @param request
- * TOOD: Requires destruction list lists endpoint.
  */
-export const destructionListCreateLoader = loginRequired(({ request }) => {
-  const searchParams = new URL(request.url).searchParams;
-  return listZaken(searchParams);
-});
+export const destructionListCreateLoader = loginRequired(
+  async ({ request }) => {
+    const searchParams = new URL(request.url).searchParams;
```

Comment:
In the old version, the query params used here were:
```
        'archiefnominatie': 'vernietigen',
        'archiefactiedatum__lt': currentDate,
```
We also wanted the already used zaken to be filtered out, so the param `not_in_destruction_list` should be used. Not sure if this was already taken into account, so I thought that I would mention it :smile:

Reply:
@SilviaAmAm nice catch, this was indeed left out. I've added this line to introduce it:

`searchParams.set("not_in_destruction_list", "true");`

Reply:
Lets discuss the other fields later on (and possibly create and issue out of it). I'll merge this to support ongoing efforts.